### PR TITLE
CEDS 1914 NACT code

### PR DESCRIPTION
**Pull request by SteveSugden**

### CODE REVIEW

File: app/uk/gov/hmrc/exports/models/declaration/ExportItem.scala [hunk 1]
```diff
@@ -63,7 +63,12 @@ object TaricCode {
   implicit val format: OFormat[TaricCode] = Json.format[TaricCode]
 }
 
-case class ItemType(nationalAdditionalCode: Seq[String], statisticalValue: String)
+case class NactCode(nactCode: String)
```
Comment (bambuchaAdm) [inline app/uk/gov/hmrc/exports/models/declaration/ExportItem.scala]: Let use full name would be less confusing for new people

Comment (PatrykRudnicki) [inline app/uk/gov/hmrc/exports/models/declaration/ExportItem.scala]: I prefer full name `NationalAdditionalCode`.

Comment (PatrykRudnicki) [reply app/uk/gov/hmrc/exports/models/declaration/ExportItem.scala]: Ok it's name from ticket

Comment (SteveSugden) [reply app/uk/gov/hmrc/exports/models/declaration/ExportItem.scala]: Its all confusing.  Whats a "TARIC" or a CUS

Comment (SteveSugden) [reply app/uk/gov/hmrc/exports/models/declaration/ExportItem.scala]: Was just about to point that out - name matches the UI

**Pull request merged**